### PR TITLE
ENH: add EL3054 to MR1/2L0

### PR DESCRIPTION
**Pull request by nrwslac**
- Added IO hardware EL 3054s to MR1/2L0
- Added corresponding PLC variables
- Renamed `FLOW` -> `FWM`
- Renamed `PRESS` -> `PRSM`
- Ran `pre-commit run --all-files`

### CODE REVIEW

File: iocBoot/ioc-lfe-optics/load_plc_databases.cmd [hunk 1]
```diff
@@ -1,3 +1,3 @@
-# Total records: 3752
-callbackSetQueueSize(9504)
+# Total records: 4083
```
Comment (klauer) [inline iocBoot/ioc-lfe-optics/load_plc_databases.cmd:1]: You can remove `load_plc_databases.cmd` - it's no longer generated (or required) in ads-ioc R0.6.0

File: lcls-plc-lfe-optics/lfe_optics/GVLs/GVL_M2L0.TcGVL [hunk 2]
```diff
@@ -3,12 +3,37 @@
...
+    {attribute 'pytmc' := '
+        pv: MR2L0:HOMS:FWM:2
+        field: EGU lpm
+        io: i
+    '}
+    fM2L0_Flow_2_val : LREAL;
+    {attribute 'TcLinkTo' := '.iRaw := TIIB[EL3054_M2L0_FWM_PRSM]^AI Standard Channel 3^Value'}
+    fM2L0_Press_1 : FB_AnalogInput;
+    {attribute 'pytmc' := '
+        pv: MR2L0:HOMS:PRSM:1
```
Comment (klauer) [inline lcls-plc-lfe-optics/lfe_optics/GVLs/GVL_M2L0.TcGVL:32]: I have to look up this acronym each time:

PRSM | Pressure Meter
-- | --

File: lcls-plc-lfe-optics/lfe_optics/GVLs/GVL_M2L0.TcGVL [hunk 3]
```diff
@@ -3,12 +3,37 @@
...
+    {attribute 'pytmc' := '
+        pv: MR2L0:HOMS:FWM:1
+        field: EGU lpm
+        io: i
+    '}
+    fM2L0_Flow_1_val : LREAL;
+    {attribute 'TcLinkTo' := '.iRaw := TIIB[EL3054_M2L0_FWM_PRSM]^AI Standard Channel 2^Value'}
+    fM2L0_Flow_2 : FB_AnalogInput;
+    {attribute 'pytmc' := '
+        pv: MR2L0:HOMS:FWM:2
```
Comment (klauer) [inline lcls-plc-lfe-optics/lfe_optics/GVLs/GVL_M2L0.TcGVL:24]: CCC:

FWM | Flow meter 4-20ma Analog
-- | --

File: lcls-plc-lfe-optics/lfe_optics/POUs/PRG_Environment.TcPOU [hunk 4]
```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+  <POU Name="PRG_Environment" Id="{ba0070ff-b060-40ec-9097-5e8efb1576b9}" SpecialFunc="None">
+    <Declaration><![CDATA[PROGRAM PRG_Environment
+VAR
+END_VAR
+]]></Declaration>
+    <Implementation>
+      <ST><![CDATA[    // MR1L0 Flow Pressure Sensors
+    GVL_M1L0.fM1L0_Flow_1(iTermBits:=12, fTermMax:=5.0427, fTermMin:=0.050472);
```
Comment (klauer) [inline lcls-plc-lfe-optics/lfe_optics/POUs/PRG_Environment.TcPOU:10]: I'll trust you on these values here - copy/paste-wise, all looks good

File: iocBoot/ioc-lfe-optics/Makefile [hunk 5]
```diff
@@ -1,4 +1,4 @@
-IOC_TOP = /reg/g/pcds/epics/ioc/common/ads-ioc/R0.4.1
+IOC_TOP = /reg/g/pcds/epics/ioc/common/ads-ioc/R0.6.0
```
Comment (klauer) [inline iocBoot/ioc-lfe-optics/Makefile:1]: Great to see the upgrade and the style fixes! 👏

**Review by klauer — APPROVED** [review]

**Review by ZLLentz — APPROVED** [review]
This looks good to me!

**Comment by nrwslac** [comment]
> This looks good to me!

merge button smash in bound.

**Pull request merged**